core/cloudflow-flink/src/main/scala/cloudflow/flink/FlinkStreamletContextImpl.scala

```diff
@@ -88,4 +89,10 @@ class FlinkStreamletContextImpl(
       )
     )
   }
+
+  private def groupId[T](inlet: CodecInlet[T]) = {
```

Comment:
This can likely also move to `StreamletContext`. (and use `SavepointPath` to get the right value)